RSSReader-Maruyama/DataManager/UserDataManager.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  LoginChecker.swift
+//  RSSReader-Maruyama
+//
+//  Created by 丸山翔 on 2020/09/03.
+//  Copyright © 2020 丸山翔. All rights reserved.
+//
+
+import UIKit
+
+class UserDataManager: NSObject {
+    
+    private let UD_USER_DATA_KEY = "userdata"
+    lazy var userDefault = UserDefaults.standard
+
+    
+    func isLoggedIn() -> Bool {
+        let user = userDefault.object(forKey: UD_USER_DATA_KEY ) as? Data
+        
+        if user != nil {
+            return true
+        } else {
+            return false
+        }
+    }
+    
+    func getUserData() -> User {
```

Comment:
```suggestion
    func load() -> User? {
```
保存がsaveなら読み取りはloadの方が適切では無いでしょうか。
また、ユーザーデータが端末に保存されているとは限られないので、返り値はOptionalにしておきましょう。

Reply:
修正しました。